pkg/controller/operconfig/cluster.go

```diff
@@ -50,14 +51,20 @@ func (r *ReconcileOperConfig) MergeClusterConfig(ctx context.Context, operConfig
 	// If there are changes to the "downstream" networkconfig, commit it back
 	// to the apiserver
 	log.Println("WARNING: Network.operator.openshift.io has fields being overwritten by Network.config.openshift.io configuration")
-	// Have to restore the typemeta due to some weird shared cache bug --cdc
+	if updateErr := r.UpdateOperConfig(operConfig); updateErr != nil {
+		errors.Wrap(err, updateErr.Error())
+	}
+	return nil
+}
+
+func (r *ReconcileOperConfig) UpdateOperConfig(operConfig *operv1.Network) error {
 	operConfig.TypeMeta = metav1.TypeMeta{APIVersion: operv1.GroupVersion.String(), Kind: "Network"}
 	us, err := k8sutil.ToUnstructured(operConfig)
 	if err != nil {
-		return errors.Wrapf(err, "failed to transmute operator config")
```

Comment:
minor nit: I doubt we want to swallow `err` here - wrapping it is useful for debugging. You can also do `fmt.Errorf("%w"` to wrap the error.

Reply:
I am not swallowing the error. `UpdateOperConfig` returns its own error and is then `errors.Wrapf`-ed with any eventual previous error, on L55

Reply:
Right, but here in line 64 you're returning a new error without the value of `err` from line 62. (And the same on line 66/67)

Reply:
Ahh, understood now! Thanks, I will fix it